db/migrations/mysql/000042_create_file_info.up.sql

```diff
@@ -0,0 +1,113 @@
+CREATE TABLE IF NOT EXISTS FileInfo (
+    Id varchar(26) NOT NULL,
+    CreatorId varchar(26) DEFAULT NULL,
+    PostId varchar(26) DEFAULT NULL,
+    CreateAt bigint(20) DEFAULT NULL,
+    UpdateAt bigint(20) DEFAULT NULL,
+    DeleteAt bigint(20) DEFAULT NULL,
+    Path text,
+    ThumbnailPath text,
+    PreviewPath text,
+    Name text,
+    Extension varchar(64) DEFAULT NULL,
+    Size bigint(20) DEFAULT NULL,
+    MimeType text,
+    Width int(11) DEFAULT NULL,
+    Height int(11) DEFAULT NULL,
+    HasPreviewImage tinyint(1) DEFAULT NULL,
+    PRIMARY KEY (Id),
+    KEY idx_fileinfo_update_at (UpdateAt),
+    KEY idx_fileinfo_create_at (CreateAt),
+    KEY idx_fileinfo_delete_at (DeleteAt),
+    KEY idx_fileinfo_postid_at (PostId)
```

Comment:
Shouldn't we create these indexes as we do for `idx_fileinfo_extension_at`? Also it would make the script similar to pq script as well. And last but not least should we also drop these indexes in the `down.sql` file?

Reply:
Added drop queries to down script. Left initial index creations as is.

Reply:
I agree that we should try to be consistent both across db types and other migrations.